deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -51,6 +51,22 @@ def read_yaml(file_path):
         return yaml.safe_load(f)
 
 
+def get_list_of_content_yml(file_path, filter, list=[]):
+    """Read a YAML file from a given path, return a list with the YAML data
+    after apply filter
+
+    Args:
+        file_path (str): Path of the YAML file to be readed
+        filter (str): filder to extract some part of yaml
+        list: a list with previous information
+
+    Returns:
+       list
```

Comment:
specifies what this list contains with its type.

Reply:
Done in commit [12919](https://github.com/wazuh/wazuh-qa/pull/2627/commits/12919d62cd044a989fb0e2781e1ee9145c3b33e4)